pkg/virt-controller/watch/vmi.go

```diff
@@ -47,6 +48,8 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/utils/trace"
 
+	"kubevirt.io/kubevirt/pkg/util/migrations"
```

Comment:
Sorry for noticing this late:
Can you move the import to the proper section? In the `kubevirt.io/kubevirt/...` block below